src/eslint/package.js

```diff
@@ -9,35 +36,39 @@ module.exports = packageConf => {
     throw new Error('E_NO_FILES');
   }
 
+  if (!data.eslintConfigType) {
+    throw new Error('E_NO_ESLINT_TYPE');
+  }
+  if (!Object.keys(ESLINT_CONFIG).includes(data.eslintConfigType)) {
+    throw new Error('E_UNKNOWN_ESLINT_TYPE');
```

Comment:
Using `YError` would allow you to give some feedback in the params (`throw new YError('E_UNKNOWN_ESLINT_TYPE', Object.keys(ESLINT_CONFIG), data.eslintConfigType)`.